src/test/java/com/google/sps/data/ElectionTest.java

```diff
@@ -150,11 +150,11 @@ public void testAddVoterInfoQueryData() throws Exception {
         new JSONObject(
             "{\"election\": {\"id\": \"9999\"},"
                 + "\"contests\": [{\"type\": \"type1\",\"office\": \"officeName\","
-                + "\"candidates\": [{\"name\": \"name1\",\"party\": \"party1\"},"
-                + "{\"name\": \"name2\",\"party\": \"party2\"}]},"
+                + "\"candidates\": [{\"name\": \"name1\",\"party\": \"party1\",\"candidateUrl\": \"www.siteOne.com\"},"
+                + "{\"name\": \"name2\",\"party\": \"party2\",\"candidateUrl\": \"www.siteOne.com\"}]},"
                 + "{\"type\": \"type2\",\"office\": \"officeName\","
-                + "\"candidates\": [{\"name\": \"name1\",\"party\": \"party1\"},"
-                + "{\"name\": \"name2\",\"party\": \"party2\"}]}],"
+                + "\"candidates\": [{\"name\": \"name1\",\"party\": \"party1\",\"candidateUrl\":\"www.siteOne.com\"},"
```

Comment:
Does adding these fields affect the Election object generated?

Reply:
The actual JSON returned by voterInfoQuery returns a field for the candidateUrl so what I did is parse that field in Candidate.java and as a result if the field is not present a JSONException is thrown by Candidate.fromJSONObject.

Reply:
Its possible the JSON returned by voterInfoQuery could be missing that field so the plan is to account for that in a follow up PR.

Reply:
Can you add an extra test that ensures an exception is thrown when the field is not present then? (in the follow up PR is fine)